cmd/eksctl-anywhere/cmd/internal/commands/artifacts/download.go

```diff
@@ -58,7 +58,7 @@ func (d Download) Run(ctx context.Context) error {
 		return fmt.Errorf("downloading images: %v", err)
 	}
 
-	if err = d.BundlesImagesDownloader.Move(ctx, artifactNames(images)...); err != nil {
+	if err = d.BundlesImagesDownloader.Move(ctx, removeFromSlice(artifactNames(images), toolsImage)...); err != nil {
```

Comment:
can u remind me of why we need separate downloader for tool img and bundle images?